index.rst

```diff
@@ -1,61 +1,546 @@
-..
-  Technote content.
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+:tocdepth: 1
 
-  Use the following syntax for sections:
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Sections
-  ========
+.. sectnum::
 
-  and
+.. note::
 
-  Subsections
-  -----------
+   This document consolidates information about how to handle CSC configuration
+   and ancillary data. The focus is on providing basis for development of a
+   unified solution on handling data that is required for proper operation of
+   CSCs in view of the different requirements and use-cases.
 
-  and
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+Introduction
+============
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+The Vera Rubin Observatory control system is highly distributed, containing
+several components that must operate in consonant. Ir order to achieve reliable
+operation it is fundamental to guarantee all components are properly setup
+and configured.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+Moreover we must be able to, at any point, identify what set of configurations
```

Comment:
I think "we" in this case is "users and deployed components" - might consider saying that explicitly.